### DISCUSSION TITLE
Problemas para realizar o login

### DISCUSSION THREAD

**Discussion by helozampieri** [handreassa/prenotami#8] [Q&A]
Oi, @handreassa ! Primeiro: muito obrigada por compartilhar o script, me ajudará muito! 

Ainda tenho pouco conhecimento de python, mas segui todos os passos e infelizmente não consegui realizar o login no prenotami pelo script. 

Recebi o seguinte erro: 
`(.venv) (base) C:\Users\55419\prenotami>python __main__.py
INFO:Timestamp: 2023-07-09 22:17:50.461074 - Required files available.
None
INFO:====== WebDriver manager ======
INFO:Get LATEST chromedriver version for google-chrome 114.0.5735
INFO:Driver [C:\Users\55419\.wdm\drivers\chromedriver\win32\114.0.5735\chromedriver.exe] found in cache

DevTools listening on ws://127.0.0.1:60724/devtools/browser/46bc32c1-7dc4-4da6-b79c-5df45a95d04e
**INFO:Exception: object of type 'NoneType' has no len()
INFO:Timestamp: 2023-07-09 22:18:16.337956 - Element WlNotAvailable not found. Start filling the forms.
INFO:Exception list index out of range**
Timestamp: 2023-07-09 22:18:16.352905 - Go ahead and fill manually the rest of the process. When finished, type quit to exit the program and close the browser.`

Entendi que o script não identificou os elementos da página de login para clicar, você poderia me ajudar a solucionar isso de alguma forma ?

Super obrigada desde já!

**Comment by handreassa** [comment]
Olá @helozampieri, acredito que eles tenham mudado a estrutura de algumas páginas. Estava ausente nos últimos dias mas devo voltar a olhar para isso em breve e atualizo o repo.

**Comment by handreassa** [comment]
@helozampieri fiz uma atualização no readme relacionado a esta publicação recente do consulado: https://conscuritiba.esteri.it/it/servizi-consolari-e-visti/servizi-per-il-cittadino-italiano/altri-servizi/informativa-sui-servizi-consolari/
